.github/CONTRIBUTING.md

```diff
@@ -92,54 +106,72 @@ Use named parameters for Jenkins Pipeline Steps:
 e.g. `sh returnStdout: true, script: command`
 
 ### Do not use _with()_ operator
+
 The _with_ operator is not yet supported by Jenkins, and it must not be used or encapsulated in a @NonCPS method.
 
 ### Use _==_ operator
+
 Use Groovy’s `==` instead of Java `equals()` to avoid NullPointerExceptions. To compare the references of objects, instead of `==`, you should use `a.is(b)` [[1]](http://groovy-lang.org/style-guide.html).
 
 ### Use GStrings
+
 In Groovy, single quotes create Java Strings, and double quotes can create Java Strings or GStrings, depending if there is or not interpolation of variables [[1]](http://groovy-lang.org/style-guide.html). Using GStrings variable and string concatenation is more simple.
 
 #### Do not use curly braces {} for variables or variable.property
+
 For variables, or variable.property, drop the curly braces:
 
 e.g. `echo "[INFO] ${name} version ${version.version} is installed."`
 
 -->  `echo "[INFO] $name version $version.version is installed."`
 
-#### Use 'single quotes' for Strings and constants. 
-#### Use "double quotes" for GStrings.
-#### Use '''triple single quotes''' for multiline Strings.
-#### Use """triple double quotes""" for multiline GStrings.
-#### Use /slash/ for regular expresions.
+#### Use 'single quotes' for Strings and constants
```

Comment:
I'm asking me if this kind of rules has any value. Using double quotes for GStrings is somehow problematic, since double quotes IS GString. The other points are rather self-evident. But my comment is clearly out of scope for this PR.